integrations/nielsen-dcr/lib/index.js

```diff
@@ -164,53 +147,57 @@ NielsenDCR.prototype.heartbeat = function(assetId, position, options) {
  */
 
 NielsenDCR.prototype.getContentMetadata = function(track, type) {
-  var properties = 'properties.';
-  var assetIdProp = 'asset_id';
-  if (type) {
-    if (type === 'preroll') properties = 'properties.content.';
-    if (type === 'video') assetIdProp = 'content_asset_id';
+  var propertiesPath = 'properties.';
+  if (type && type === 'contentEvent') propertiesPath = 'properties.content.';
```

Comment:
@briemcnally Tweaked the logic to device whether we should be checking in `properties.content` for metadata. Turns out we do need this check. `contentEvent` includes a pre-roll ad event and "Video Playback Seek Completed/Buffer Completed" and "Video Playback Resumed".